app/frontend/src/features/auth/signup/Signup.tsx

```diff
@@ -1,33 +1,133 @@
-import React from "react";
-import { Redirect, Route, Switch } from "react-router-dom";
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import React, { useEffect } from "react";
+import { Link, Redirect, Route, Switch } from "react-router-dom";
 
 import Alert from "../../../components/Alert";
-import PageTitle from "../../../components/PageTitle";
-import { signupRoute } from "../../../routes";
+import AuthHeader from "../../../components/AuthHeader";
+import AuthBg from "../../../resources/auth-bg.png";
+import { loginRoute, signupRoute } from "../../../routes";
 import { useAuthContext } from "../AuthProvider";
 import CompleteSignupForm from "./CompleteSignupForm";
 import EmailForm from "./EmailForm";
 
+const useStyles = makeStyles((theme) => ({
```

Comment:
If lots of these are the same as the login page, they should be shared

Reply:
Where would you put them?

Reply:
maybe a `useAuthStyles.ts` file in features/auth?